library/cgi/escapeURIComponent_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../../spec_helper'
+require 'cgi'
+
+describe "CGI.escapeURIComponent" do
+  ruby_version_is "3.2" do
```

Comment:
**suggestion:** it's common to have the version guard top-level and `describe` nested in case it's a new method and it's introduced in this version.

Reply:
👍 No problem, I swapped it.